proto/types/action.proto

```diff
@@ -277,28 +277,47 @@ message Logs {
   repeated Log logs = 1;
 }
 
+// Deprecated
 message EvmTransfer {
   bytes amount = 1;
   string from = 2;
   string to = 3;
 }
 
+// Deprecated
 message EvmTransferList {
   repeated EvmTransfer evmTransfers = 1;
 }
 
+// Deprecated
 message ActionEvmTransfer {
   bytes actionHash = 1;
   uint64 numEvmTransfers = 2;
   repeated EvmTransfer evmTransfers = 3;
 }
 
+// Deprecated
 message BlockEvmTransfer {
   uint64 blockHeight = 1;
   uint64 numEvmTransfers = 2;
   repeated ActionEvmTransfer actionEvmTransfers = 3;
 }
 
+message ActionSystemLog {
+  message Operation {
```

Comment:
so no need for "from" and "to"?

is "address" == "to"?

Reply:
this is refer from rosetta API, they break down transaction into multiple operation. each operation contains one address and one positive/negative amount.
good part is this is very flexible, you can have 1to1 1tomany, single sided transaction.
down side is this introduce negative amount.